src/components/CombinedFeaturePanel/editors/AutoEditor.tsx

```diff
@@ -134,12 +146,55 @@ export const AutoEditor = ({ feature, tagKey }: AutoEditorProps) => {
     });
   }, []);
 
+  const handlePickerValueChange = React.useCallback(
```

Comment:
```suggestion
  const handleLanguageSuffixChange = React.useCallback(
```

Reply:
suggestion (blocking): The code here should not "know" that the subclass uses a specific UI element like a picker to change the language. It just knows that the subclass might, somehow, change the value.